### DISCUSSION TITLE
Q: 在浏览器页面直接访问评价页面提示GET https://unpkg.com/@waline/admin net::ERR_CONNECTION_TIMED_OUT || Q: Directly access the evaluation page on the browser page prompt GET https://unpkg.com/@waline/admin net::ERR_CONNECTION_TIMED_OUT

### DISCUSSION THREAD

**Discussion by gglss** [walinejs/waline#2231] [Q&A] [answered]
### Describe the bug | 问题描述

环境：
    使用LeanCloud+Vercel部署
    使用firefox浏览器，chrome浏览器访问
    

![image](https://github.com/walinejs/waline/assets/81339751/c56d8645-bd8c-4817-898a-bba5b8731b87)

### Website URL | 问题网站

https://waline.gglss.top

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

environment:
    Deploy using LeanCloud+Vercel
    Use firefox browser, chrome browser to access
    

![image](https://github.com/walinejs/waline/assets/81339751/c56d8645-bd8c-4817-898a-bba5b8731b87)

### Website URL | Problem website

https://waline.gglss.top

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
unpkg.com 的网络问题，可以参考这个方案 https://github.com/walinejs/waline/issues/776 切换其他地址

**Reply by gglss** [reply]
我使用了管理员地址[#552](https://github.com/walinejs/waline/issues/552)，但是我这边没有进行管理员注册，在那个[#552](https://github.com/walinejs/waline/issues/552)登录注册后就显示401没有授权，导致那个管理地址也没有办法使用

**Reply by gglss** [reply]
我这边注册页面打不开，和上面写的这个问题一样

---
I can’t open the registration page here, it’s the same problem as mentioned above
